packages/lambdas/package.json

```diff
@@ -26,7 +26,7 @@
     "@medplum/fhirtypes": "^2.0.32",
     "@metriport/api-sdk": "^7.1.3-alpha.10",
     "@metriport/commonwell-sdk": "^4.8.0",
-    "@metriport/core": "^1.3.0-alpha.31",
+    "@metriport/core": "1.3.0-alpha.1",
```

Comment:
is this version downgrade intentional?

Reply:
Yeah, diff branches releasing diff alpha versions... one startes at 0 and the other one at 30 😀